### PR TITLE
Seen no longer includes anything after first space

### PR DESCRIPTION
**Pull request by J3RN**
"seen" used to include all characters. For example, ".seen J3RN         "
would search for "j3rn         ". Also, ".seen not a real nick" would
seach for "not a real nick". Now, the first example will search for
"j3rn" and the latter for "not".

### CODE REVIEW

File: modules/seen.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def f_seen(jenni, input):
 
     if not input.group(2):
         return jenni.say('Please provide a nick.')
-    nick = input.group(2).lower()
+    nick = input.group(2).lower().split(' ')[0]
```
Comment (kaneda) [inline modules/seen.py]: It seems like we could just change the regex to split on \w (word boundary) and group two could be correct

**Comment by J3RN** [comment]
@kaneda I switched from splitting on the default regex to a more specialized regex that only captures the first word. I hope this is what you were looking for.

**Comment by kaneda** [comment]
@J3RN that's great, ty. Test and squash it, then I'll merge down.

**Comment by J3RN** [comment]
@kaneda I've manually tested this. From what I can tell, there's no formal testing in this codebase (unless I'm missing something?).

Also squashed.

**Comment by kaneda** [comment]
Nope, that's it, thanks @J3RN

**Pull request merged**